js_modules/dagit/packages/core/src/asset-graph/AssetRunLinking.tsx

```diff
@@ -37,6 +37,10 @@ export const AssetRunLink: React.FC<{
     target="_blank"
     rel="noreferrer"
   >
-    {children || <CaptionMono>{titleForRun({id: runId})}</CaptionMono>}
+    {children || (
+      <span style={{fontSize: '1.2em', fontFamily: FontFamily.monospace}}>
```

Comment:
Should we be doing this more generally across the app?

Reply:
I think we should definitely consider it!